pallets/thea/src/lib.rs

```diff
@@ -498,7 +498,6 @@ pub mod pallet {
 		) -> DispatchResult {
 			let user = ensure_signed(origin)?;
 			// Put a soft limit of size of beneficiary vector to avoid spam
-			ensure!(beneficiary.len() <= 100, Error::<T>::BeneficiaryTooLong);
```

Comment:
Please remove comment as well if we don't need this assertion.

Reply:
done